packages/react/src/redux.ts

```diff
@@ -112,52 +112,63 @@ function createReduxEnhancer(enhancerOptions?: Partial<SentryEnhancerOptions>):
+          }
 
-        /* Set latest state to scope */
-        const transformedState = options.stateTransformer(newState);
-        if (typeof transformedState !== 'undefined' && transformedState !== null) {
-          const client = getClient();
-          const options = client?.getOptions();
-          const normalizationDepth = options?.normalizeDepth || 3; // default state normalization depth to 3
-
-          // Set the normalization depth of the redux state to the configured `normalizeDepth` option or a sane number as a fallback
-          const newStateContext = { state: { type: 'redux', value: transformedState } };
-          addNonEnumerableProperty(
-            newStateContext,
-            '__sentry_override_normalization_depth__',
-            3 + // 3 layers for `state.value.transformedState`
-              normalizationDepth, // rest for the actual state
-          );
-
-          scope.setContext('state', newStateContext);
-        } else {
-          scope.setContext('state', null);
-        }
+          /* Set latest state to scope */
+          const transformedState = options.stateTransformer(newState);
+          if (typeof transformedState !== 'undefined' && transformedState !== null) {
+            const client = getClient();
+            const options = client?.getOptions();
+            const normalizationDepth = options?.normalizeDepth || 3; // default state normalization depth to 3
+
+            // Set the normalization depth of the redux state to the configured `normalizeDepth` option or a sane number as a fallback
+            const newStateContext = { state: { type: 'redux', value: transformedState } };
+            addNonEnumerableProperty(
+              newStateContext,
+              '__sentry_override_normalization_depth__',
+              3 + // 3 layers for `state.value.transformedState`
+                normalizationDepth, // rest for the actual state
+            );
+
+            scope.setContext('state', newStateContext);
+          } else {
+            scope.setContext('state', null);
+          }
 
-        /* Allow user to configure scope with latest state */
-        const { configureScopeWithState } = options;
-        if (typeof configureScopeWithState === 'function') {
-          configureScopeWithState(scope, newState);
-        }
+          /* Allow user to configure scope with latest state */
+          const { configureScopeWithState } = options;
+          if (typeof configureScopeWithState === 'function') {
+            configureScopeWithState(scope, newState);
+          }
+
+          return newState;
+        };
+      }
 
-        return newState;
-      };
+      const store = next(sentryWrapReducer(reducer), initialState);
+
+      // eslint-disable-next-line @typescript-eslint/unbound-method
+      store.replaceReducer = new Proxy(store.replaceReducer, {
+        apply: function (target, thisArg, args) {
+          target.apply(thisArg, [sentryWrapReducer(args[0])]);
```

Comment:
Just saw your question here: https://github.com/LeComptoirDesPharmacies/sentry-javascript/commit/631f71e61748c1b89cd7af479328ce6a478794da

> I am wondering if we can do better than accessing the index [0] directly ?
No really as we need to use the first argument :/